src/EditorFeatures/CSharpTest/Microsoft.CodeAnalysis.CSharp.EditorFeatures.UnitTests.csproj

```diff
@@ -52,13 +52,15 @@
   </ItemGroup>
   <ItemGroup>
     <Compile Remove="Debugging\Resources\*.*" />
+    <Compile Remove="Diagnostics\NamingStyles\**" />
```

Comment:
are the changes to the csproj necessary?

Reply:
no.  but i noticed this empty folder that was annoying me :)